src/help.html

```diff
@@ -127,172 +138,322 @@ <h2>Shipping FAQs</h2>
-          loading="lazy"
-          src="https://www.youtube.com/embed/6VnczBUTtNY"
-          title="YouTube video showing how to upload music on to the BeakBox"
-          frameborder="0"
-          allow="autoplay; clipboard-write; encrypted-media; picture-in-picture; web-share"
-          allowfullscreen
-        ></iframe>
-      </div>
+    <div class="flex-container list-style-decimal">
+      <h3>How to rename a playlist</h3>
+      <ul>
+        <li>Choose the playlist you wish to rename.</li>
+        <li>Click the "Edit" icon.</li>
 
-      <div class="notecard">
-        <p>Please make sure that the music files you’re uploading onto the microSD card are in .mp3 file format and are dropped (or pasted) directly to the root folder.</p>
-      </div>
-      <p>Steps to create a custom playlist:</p>
-      <ol>
-        <li>Unplug your BeakBox</li>
-        <li>Remove the microSD card from the BeakBox by pulling it out</li>
-        <li>Insert the microSD card into the SD card adapter provided with the BeakBox</li>
-        <li>Insert the adapter into the SD card slot on your computer</li>
-        <li>Open the microSD card root folder in your file explorer and drag and drop the tunes you want to upload onto your BeakBox</li>
-        <li>Eject the microSD card safely from your computer, take it out and insert it back into the BeakBox</li>
-        <li>Plug it in and enjoy new tunes!</li>
-      </ol>
-      <div class="notecard flex-container">
-        <p>
-          We recommend having only music files on your microSD card. BeakBox has to scan through the files and folders on the card looking for mp3 files and it can take long time if there are a lot of
-          non-mp3 files.
-        </p>
         <figure>
-          <img src="/imgs/help-no-folders.png" alt="" />
+          <img src="/imgs/edit-playlist.png" alt="Modal highlighting the edit icon">
         </figure>
-      </div>
-      <div class="notecard">
-        <p>BeakBox can play up to 100 songs, please do not upload more than that.</p>
-      </div>
-      <h3>How to encourage my bird to use the BeakBox?</h3>
-      <p>
-        We have a blog article sharing <a href="https://mybeakbox.com/blog/6-tips-to-get-the-most-out-of-your-beakbox/" target="_blank">the best tips and tricks</a> that will help your bird get used
-        to the BeakBox and get the most out of it.
-      </p>
+       
+        <li>A prompt will appear, allowing you to set the new name for the playlist. Enter the desired name and click "OK".</li>
+        
+        <figure>
+          <img src="/imgs/set-new-playlist-name.png" alt="Modal showing the interface for entering a new name for the playlist">
+        </figure>
+
+      </ul>
+      <p>Your playlist should now be renamed.</p>
     </div>
-  </article>
-</section>
 
-<section>
-  <article class="main-container layout-grid">
-    <div class="flex-container">
-      <h2>Troubleshooting</h2>
+    <div class="flex-container list-style-decimal">
```

Comment:
The `list-style-decimal` seems like it would make more sense on the actual `ul` tag?

Reply:
Upon further investigation, this class is not used in the CSS and can be removed from the HTML